docs/src/pages/components/slider/InputSlider.tsx

```diff
@@ -16,10 +16,7 @@ export default function InputSlider() {
     30,
   );
 
-  const handleSliderChange = (
-    event: React.SyntheticEvent,
-    newValue: number | number[],
-  ) => {
+  const handleSliderChange = (event: unknown, newValue: number | number[]) => {
```

Comment:
Why change these?

Reply:
I was hesitating to use `any` instead of `unknown`. The incentive is that the event is useless in the **nominal** case, developers don't need it.

In HEAD the correct type is:

```suggestion
  const handleSliderChange = (event: React.SyntheticEvent | Event, newValue: number | number[]) => {
```

In this PR, the correct type is

```suggestion
  const handleSliderChange = (event: Event, newValue: number | number[]) => {
```

Do you have a preference?

Reply:
My preference is keeping the current behavior and not cramming as much unrelated changes into a PR as possible. We documented the event type before, we should keep documenting it.

Reply:
Ok